.claude/scripts/fetch_github_pr.py

```
@@ -0,0 +1,346 @@
+        comment_id = target_comment['id']
+
+        print(f"Found comment {comment_id}", file=sys.stderr)
+
+        # Parse comment
+        draft = parse_qe_comment(comment_body, pr_title, pr_body)
+
+        # Output
+        if args.output:
+            output_file = args.output
+        else:
+            output_dir = '/tmp/test-specs/drafts'
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/test-specs/drafts"

(S108)